lib/utilities/extensions.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex/utilities/colors.dart';
+import 'package:intl/intl.dart';
+
+extension StringExt on String {
+  String get capitalize => '${this[0].toUpperCase()}${substring(1).toLowerCase()}';
+
+  String get toTitleCase =>
+      this.toLowerCase().split(' ').map((word) => word[0].toUpperCase() + word.substring(1)).join(' ');
+
+  String get pokemonImage {
+    // return 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/$this.png';
```

Comment:
delete comment

Reply:
i commented that for the other artwork so that it's available if i want to use it. is it not ideal to leave comments for those kinds of reasons? HAHAHA

Reply:
HAHA for personal project, its okay but since other people are reviewing your code, they might not get why the code is commented out